api/src/main/java/org/pmiops/workbench/cohortreview/AnnotationQueryBuilder.java

```diff
@@ -239,7 +258,7 @@ private String getSql(CohortReview cohortReview, List<CohortStatus> statusFilter
           if (obj != null) {
             String column = columns.get(i);
             if (column.equals(REVIEW_STATUS_COLUMN)) {
-              result.put(column, StorageEnums.cohortStatusFromStorage((Short) obj).name());
+              result.put(column, StorageEnums.cohortStatusFromStorage(((Number) obj).shortValue()).name());
```

Comment:
Regression test?

Reply:
We already have tests in AnnotationQueryBuilder that test this logic... but they use H2 / in-memory which works differently than MySQL apparently, because they pass (whereas enum annotations weren't working for MySQL in test.)

The only way of testing this would be to spin up MySQL, which seems like more trouble than it's worth.